advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -168,19 +171,26 @@ private fun createRequest(pkg: Package): VulnerabilitiesForPackageRequest? {
 }
 
 private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.Vulnerability {
-    val (scoringSystem, severity) = severity.firstOrNull()?.let {
+    var (scoringSystem, score) = severity.firstOrNull()?.let {
```

Comment:
IMO the variable should stay called "severity", esp. as that's also the field name of `databaseSpecific` you're reading out below.